src/zoo/owl_zoo_ver.ml

```diff
@@ -61,13 +61,14 @@ let get_remote_vid (gid : string) =
 
 (** Get the latest version downloaded on local machine; if the local version is
 not found on record, get the newest vid from Gist server. *)
-let get_latest_vid (gid : string) =
-  let v, _, _, miss_flag = get_value gid in
-  if (miss_flag == false) then (
+let get_latest_vid (gid : string) (tol : float)=
```

Comment:
can you be a bit careful about the format? Try to avoid letting others fix these small issues for you all the time. Add space between ) and = please.